wolvic/wolvic_permission_manager.cc

```diff
@@ -363,6 +363,10 @@ blink::mojom::PermissionStatus WolvicPermissionManager::GetPermissionStatus(
     blink::PermissionType permission,
     const GURL& requesting_origin,
     const GURL& embedding_origin) {
+  if (permission == blink::PermissionType::PAYMENT_HANDLER) {
+    return blink::mojom::PermissionStatus::GRANTED;
+  }
```

Comment:
Hmm I think we should still ask for this permission anyway

Reply:
We can, I guess, but bear in mind that Chrome Desktop doesn't do it.

Reply:
As a matter of fact, the Chromium code that handles the web payment method manifest downloading expect the permission to be GRANTED by default. I don't think we can ask the user about this.